### PR TITLE
Blog app - controllers

### PR DESCRIPTION
**Pull request by tugsem**
## In this branch I :
- Used controllers to handle requests and render empty views.
- Understood Rails naming conventions.
- Used params from browser requests in a safe way.

### CODE REVIEW

File: db/migrate/20220823180758_remove_posts_counter_from_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class RemovePostsCounterFromUsers < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :users, :postsCounter, :integer
+  end
+end
```
Comment (andersonlebon) [inline db/migrate/20220823180758_remove_posts_counter_from_users.rb:5]: ## [OPTIONAL]

- [ ] Kindly solve these errors in your migration. You are removing columns that do not exist. 

image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
Hi @tugsem 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly make sure your README file follows the microverse README template, thus it should contain all the required sections just like in the Microverse README file.

image

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220823180758_remove_posts_counter_from_users.rb (hunk 1 above)
Comment (tugsem) [reply db/migrate/20220823180758_remove_posts_counter_from_users.rb:5]: Yes, the reason is I delete the files after I remove columns.

**Review by descholar-ceo — APPROVED** [review]
Hi @tugsem 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 You have implemented the changes requested on the previous review ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omarsalem7** [comment]
Hi @tugsem 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**